vendor/Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: all
+all: unzip verify headers
+
+.PHONY: unzip
+unzip:
+	./unzip.sh
+
+lib/libguile-3.0.a: big_sur/guile/3.0.7_2/lib/libguile-3.0.a arm64_big_sur/guile/3.0.7_2/lib/libguile-3.0.a
```

Comment:
TODO probably want to work out a smarter way of finding the `.a` files.  These paths will change when versions change.